src/kitconcept/seo/serializers/summary.py

```diff
@@ -0,0 +1,10 @@
+from plone.restapi.interfaces import IJSONSummarySerializerMetadata
+from zope.interface import implementer
+
+
+@implementer(IJSONSummarySerializerMetadata)
+class JSONSummarySerializerMetadata:
+    """Additional metadata to be exposed on listings."""
+
+    def default_metadata_fields(self):
+        return {"image_field", "image_scales", "effective", "Subject"}
```

Comment:
@sneridagh Why does this belong in kitconcept.seo? These should be added in plone.volto (the first 2 already are) if we need them for listings.

Reply:
Uh, this comes from cookieplone templates: https://github.com/plone/cookieplone-templates/blob/main/backend_addon/%7B%7B%20cookiecutter.__folder_name%20%7D%7D/src/%7B%7B%20cookiecutter.__package_namespace%20%7D%7D/%7B%7B%20cookiecutter.__package_name%20%7D%7D/serializers/summary.py

@ericof why is that in there? Should we remove it?

Reply:
@davisagli removing it for now